packages/inventory/src/Stocktake/api/hooks/document/useStocktake.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  useParams,
+  UseQueryResult,
+  useQuery,
+} from '@openmsupply-client/common';
+import { StocktakeFragment } from '../../operations.generated';
+import { useStocktakeApi } from '../utils/useStocktakeApi';
+
+export const useStocktakeNumber = () => {
```

Comment:
yes, I know. "each hook in its own file". There's a couple of instances of helper functions which are shared by a couple of hooks - they aren't used outside of the ./hooks folder though so I haven't split into their own file. simply exported in order to share

Reply:
Think its ok to keep things together if they only used locally. Putting it in a separate file might actually harm more than it is good in such cases... Maybe just add a comment?